app/components/MenuPopup/MenuPopup.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import * as Popover from "@radix-ui/react-popover";
+import { DotsHorizontalIcon, Cross2Icon } from "@radix-ui/react-icons";
```

Comment:
use react-icons lib here and please remove the @radiix/reactui lib from the repo using yarn

Reply:
Made the above mentioned changes and removed radix colors and icons as well @UmmarHamid